back_end/RayTracer/Views/Index/Index.aspx

```diff
@@ -0,0 +1,60 @@
+﻿<%@ Page Language="C#" Inherits="System.Web.Mvc.ViewPage" %>
```

Comment:
This seems like a testing file?

Reply:
Yeh you're right. Just got rid of it! Thanks :)